packages/cli/src/commands/start.ts

```diff
@@ -65,6 +66,12 @@ export class Start extends BaseCommand {
 
 	protected server = new Server();
 
+	constructor(argv: string[], cmdConfig: IConfig) {
+		super(argv, cmdConfig);
+		this.setInstanceType('main');
```

Comment:
when I suggested using an abstract property, I meant that we can then overwrite the property in the derived class without having to override the constructor like this.
but for now this work fine.